app/src/main/res/layout/item_status.xml

```diff
@@ -87,71 +87,43 @@
             android:textSize="?attr/status_text_medium"
             tools:text="13:37" />
 
-    </RelativeLayout>
-
-    <com.keylesspalace.tusky.view.FlowLayout
-        android:id="@+id/status_content_warning_bar"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@+id/status_name_bar"
-        android:layout_marginBottom="4dp"
-        android:layout_toEndOf="@+id/status_avatar"
-        android:focusable="true"
-        android:visibility="gone"
-        app:paddingHorizontal="4dp">
-
         <TextView
             android:id="@+id/status_content_warning_description"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
+            android:layout_below="@id/status_display_name"
             android:lineSpacingMultiplier="1.1"
             android:textColor="?android:textColorPrimary"
+            android:visibility="gone"
             android:textSize="?attr/status_text_medium" />
 
-        <RelativeLayout
-            android:id="@+id/status_content_warning_button_area"
+        <ToggleButton
+            android:id="@+id/status_content_warning_button"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_below="@+id/status_content_warning_description"
-            android:layout_marginBottom="4dp"
-            android:layout_toEndOf="@+id/status_avatar"
-            android:focusable="true"
-            android:paddingTop="6dp"
+            android:layout_centerHorizontal="true"
+            android:background="?attr/content_warning_button"
+            android:minHeight="0dp"
+            android:minWidth="0dp"
```

Comment:
I think a `minWidth` of about 150dp would help so the button stays the same size when the text changes (at least for most languages).

Reply:
This is just moved from the old version - do you still want me to change it?

Reply:
yes please